backend/README.md:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
Same as comment about the .gitignore.txt Zone Identifier (re-typed below):
"You may want to add this to the .gitignore, I'm not sure it needs to be added to the GitHub. Unless you need it consider adding *.Zone.Identifier to the .gitignore and it should prevent these streams from being added."